### PR TITLE
Add annual report for SIG-MC

### LINKED ISSUE #5505: 2021 Annual Report: SIG Multicluster
Chairs: @jeremyot @pmorie 
Liasion: @parispittman

- [ ] Consult SIG to complete draft of report
- [ ] If needed, consult with Steering Committee liasion on any private issues or concerns that arrise while completing report
- [ ] Submit PR to git.k8s.io/community repo with copy of report, assigning your SC liasion for review
- [ ] Steering Committee reviews report, and may make comments/ask questions
- [ ] Once all comments are addressed, SC will approve report to merge
- [ ] If needed, any follow up items may be brought to the "Chairs, Tech Leads, and Organizers" meeting in April

Once all the above items are complete, this issue may be `/close`'d

Key dates:
- Initial PR to communtiy repo should be opened by March 1st, 2021
- PR should be reviewed and merged by April 1st, 2021

More detailed information on the annual reports process is available [here](https://git.k8s.io/community/committee-steering/governance/annual-reports.md).

If you have any questions or concerns about this process, you may reach the Steering Committee via the following methods:
- Slack/E-mail your liasion directly
- The public #steering-committee channel in Slack
- The public mailing list steering@kubernetes.io
- The private mailing list steering-private@kubernetes.io for any private or sensitive issues.

/sig multicluster
/committee steering

### PR DESCRIPTION
**Pull request by JeremyOT**
**Which issue(s) this PR fixes**:

Fixes #5505

### CODE REVIEW

**Review by makkes — APPROVED** [review]

**Review by hectorj2f — APPROVED** [review]

**Review by parispittman — CHANGES_REQUESTED** [review]

File: sig-multicluster/annual-report-2021.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+* Is your README accurate? have a CONTRIBUTING.md file?
+  * Yes, our README is accurate: https://github.com/kubernetes/community/blob/master/sig-multicluster/README.md
+  * Our CONTRIBUTING.md file is up to date: https://github.com/kubernetes/community/blob/master/sig-multicluster/CONTRIBUTING.md
+
+* All subprojects correctly mapped and listed in sigs.yaml?
+  * Yes
+
+* What’s your meeting culture? Large/small, active/quiet, learnings? Meeting notes up to date? Are you keeping recordings up to date/trends in community members watching recordings?
+  * SIG Multicluster holds regular weekly meetings which are normally well attended with ~10 participants.
+  * Due to the increase in activity in 2020, SIG meetings went from bi-weekly to weekly and almost always have a solid agenda.
```
[the remark this answers is not in the document]
Comment (pmorie) [reply sig-multicluster/annual-report-2021.md:18]: That's a really good question and I'm not sure there's a clear answer, here are some factors that may be related:

- Calls for participation during Kubecon
- More focused work on individual problems (cluster ID, multicluster services, work) vs large-scoped efforts (federation)

File: sig-multicluster/annual-report-2021.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  * Meetings are recorded and kept up to date.
+  * We'd love to see more participation, more contributors bringing their perspectives and use cases.
```
[the remark this answers is not in the document]
Comment (pmorie) [reply sig-multicluster/annual-report-2021.md:20]: Personally, I think we have capacity for folks to help moving any of our focused efforts (mentioned in above comment) forward in terms of hands-on code or technical writing.

File: sig-multicluster/annual-report-2021.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  * [Work API](https://docs.google.com/document/d/1cWcdB40pGg3KS1eSyb9Q6SIRvWVI8dEjFp9RI0Gk0vg) - [sigs.k8s.io/work-api](https://sigs.k8s.io/work-api)
+  * MCS API [KEP-1645](https://github.com/kubernetes/enhancements/tree/master/keps/sig-multicluster/1645-multi-cluster-services-api) - [sigs.k8s.io/mcs-api](https://sigs.k8s.io/mcs-api)
+  * Cluster ID [KEP-2149](https://github.com/kubernetes/enhancements/tree/master/keps/sig-multicluster/2149-clusterid)
+* Year to date KEP work review: What’s now stable? Beta? Alpha? Road to alpha?
+  * Kubefed: nearly Beta.
+  * Work API: nascent, on the road to alpha.
+  * MCS API: nearly Beta.
+  * Cluster ID: nascent, on the road to alpha.
+* What areas and/or subprojects does the group need the most help with?
+  * SIG MC needs help with all of the above - especially when it comes to use cases and validating our approaches for different environments and deployment models.
```
[the remark this answers is not in the document]
Comment (JeremyOT) [reply sig-multicluster/annual-report-2021.md:63]: We haven't and could use the help!

File: sig-multicluster/annual-report-2021.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+  * Yes, our chairs are from two different companies. We have regular participation in SIG projects from those companies and several others. We'd like to see additional users and companies bring their use cases and contribute to solutions.
+
+## Current initiatives and project health
+* What are initiatives that should be highlighted, lauded, shout out, that your group is proud of? Currently underway? What are some of the longer tail projects that your group is working on?
+  * [Kubefed](http://sigs.k8s.io/kubefed) - long running subproject with active contribution.
+  * [Work API](https://docs.google.com/document/d/1cWcdB40pGg3KS1eSyb9Q6SIRvWVI8dEjFp9RI0Gk0vg) - [sigs.k8s.io/work-api](https://sigs.k8s.io/work-api)
+  * MCS API [KEP-1645](https://github.com/kubernetes/enhancements/tree/master/keps/sig-multicluster/1645-multi-cluster-services-api) - [sigs.k8s.io/mcs-api](https://sigs.k8s.io/mcs-api)
+  * Cluster ID [KEP-2149](https://github.com/kubernetes/enhancements/tree/master/keps/sig-multicluster/2149-clusterid)
+* Year to date KEP work review: What’s now stable? Beta? Alpha? Road to alpha?
```
[the remark this answers is not in the document]
Comment (JeremyOT) [reply sig-multicluster/annual-report-2021.md:57]: Added timeframes

File: sig-multicluster/annual-report-2021.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+  * We are actively soliciting new ideas and looking for new people to drive them.
+* What programs do you participate in for new contributors?
+  * At Kubecon we use the contributor track to give insight into what the SIG is working on and ask for new contributors. We've seen growth in weekly meeting attendance and new presenters as a result.
+  * When in-person Kubecons return we'll have Meet & Greets for new contributors (and everyone we didn't see in 2020!)
+* Does the group have contributors from multiple companies/affiliations? Can end users/companies contribute in some way that they currently are not?
+  * Yes, our chairs are from two different companies. We have regular participation in SIG projects from those companies and several others. We'd like to see additional users and companies bring their use cases and contribute to solutions.
+
+## Current initiatives and project health
+* What are initiatives that should be highlighted, lauded, shout out, that your group is proud of? Currently underway? What are some of the longer tail projects that your group is working on?
+  * [Kubefed](http://sigs.k8s.io/kubefed) - long running subproject with active contribution.
```
[the remark this answers is not in the document]
Comment (JeremyOT) [reply sig-multicluster/annual-report-2021.md]: Added the main 2020 Kubefed accomplishment.

File: sig-multicluster/annual-report-2021.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+  * [Work API](https://docs.google.com/document/d/1cWcdB40pGg3KS1eSyb9Q6SIRvWVI8dEjFp9RI0Gk0vg) - [sigs.k8s.io/work-api](https://sigs.k8s.io/work-api)
```
[the remark this answers is not in the document]
Comment (JeremyOT) [reply sig-multicluster/annual-report-2021.md:54]: Right now, getting to beta is the primary goal for Kubefed in 2021

File: sig-multicluster/annual-report-2021.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+  * Both SIG chairs are active and attend most SIG meetings.
+* How do you measure membership? By mailing list members, OWNERs, or something else?
+  * Mailing list members for general interest. Slack, weekly meeting attendance, and messages to the list for active engagement.
+* How does the group measure reviewer and approver bandwidth? Do you need help in any area now? What are you doing about it?
+  * The SIG tracks progress at weekly meetings and tends to closely watch and report on PRs and progress as a group so reviewer/approver bandwidth is not an issue.
+  * Kubefed tends to operate independently. Reviewer bandwidth has not been an issue, questions on PRs and Slack tend to get relatively quick response.
+* Is there a healthy onboarding and growth path for contributors in your SIG? What are some activities that the group does to encourage this? What programs are you participating in to grow contributors throughout the contributor ladder?
+  * This SIG is in growth mode, and tries to create room for new ideas and projects driven by new and existing members alike. In 2020 we've seen two new initiatives kick off driven by relatively new members.
```
[the remark this answers is not in the document]
Comment (JeremyOT) [reply sig-multicluster/annual-report-2021.md]: Good question - elaborated a bit more

File: sig-multicluster/annual-report-2021.md [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+* How does the group get updates, reports, or feedback from subprojects? Are there any springing up or being retired? Are OWNERS.md files up to date in these areas?
+  * Most group updates are at the weekly SIG meeting, big changes/ideas are shared with the list.
+* Same question as above but for working groups.
+  * Working groups join SIG weekly to discuss major updates, we don’t have separate meetings. Most WG discussion is on slack.
+* When was your last monthly community-wide update? (provide link to deck and/or recording)
+  * Our last community wide update was 2020-09-17: [SIG MC Community Update Slides](https://docs.google.com/presentation/d/1WKtsiSQn0sQ3IaSql4pGnH8Qt9ibBjHXMFBYZBhWeII).
+
+## Membership
+* Are all listed SIG leaders (chairs, tech leads, and subproject owners) active?
+  * Both SIG chairs are active and attend most SIG meetings.
```
[the remark this answers is not in the document]
Comment (JeremyOT) [reply sig-multicluster/annual-report-2021.md:34]: We don't have other named roles, but I added a separate line to acknowledge Kubefed owners as they tend to operate mostly independently and prefer slack to the weekly meeting

File: sig-multicluster/annual-report-2021.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (JeremyOT) [reply sig-multicluster/annual-report-2021.md:18]: Good points, pulled this into the doc.

File: sig-multicluster/annual-report-2021.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (JeremyOT) [reply sig-multicluster/annual-report-2021.md]: Made a node in the doc

**Comment by parispittman** [comment]
/lgtm with steering hat on 

pull the hold in ~24 hours or so for last comments

**Comment by JeremyOT** [comment]
Looks like there are no new comments. Thanks all!

/hold cancel

**Pull request merged**